code/src/components/Thoughts.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { formatDistance } from 'date-fns';
+
+
+const Thoughts = ({ loading, thoughts, onNewLikeSubmit }) => {
+
+    if (loading) {
+        return <h4 className="loading-text">Happy tweets incoming in 3.. 2.. 1..</h4>
+    }
+
+    return (
+        <section className='thoughts-container'>
+            {thoughts.map((thought) => ( 
+                <article className='thoughts-cards' key={thought._id}>
+                    <h4 className='thoughts-message'>{thought.message}</h4>
+                    <div className='time-and-like-container'>
+                        <section className='like-container'>
+                            <button
+                                className={thought.hearts === 0 ? 'no-like-btn' : 'like-btn'}
```

Comment:
Ternary operator on the 0 likes button!!! I wish I implemented that in my project. Suuuper nice attention to detail and user experience.